proto/cosmos/accounts/vesting/query.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+package cosmos.accounts.vesting;
+
+import "cosmos/accounts/vesting/vesting.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "cosmossdk.io/x/accounts/vesting/types";
+
+// QueryVestingAccountInfoRequest get vesting account info
+message QueryVestingAccountInfoRequest {}
+
+// QueryVestingAccountInfoResponse return vesting account info
+message QueryVestingAccountInfoResponse {
+  // original_vesting defines the value of the account original vesting.
+  repeated cosmos.base.v1beta1.Coin original_vesting = 1
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+
+  // delegated_free defines the value of the account delegated free.
+  repeated cosmos.base.v1beta1.Coin delegated_free = 2
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
+
+  // delegated_free defines the value of the account delegated free.
+  repeated cosmos.base.v1beta1.Coin delegated_vesting = 3
+      [(gogoproto.nullable) = false, (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"];
```

Comment:
Wrong comment for this field. Also, these field comments do not actually say anything.